demos/guru_scripts/docker/tutorial/3.x/gsql102/queries/multiple-hop1.gsql

```diff
@@ -0,0 +1,14 @@
+USE GRAPH ldbc_snb
+SET syntax_version="v2"
+
+INTERPRET QUERY () FOR GRAPH ldbc_snb {
+
+  TagClass1 = {TagClass.*};
+
+  TagClass2 =  SELECT t
```

Comment:
please read gsql 102 of 3.0 doc, this query has changed. We can start from TagClass directly.